lib/core/odp/odp_event_manager.ts

```diff
@@ -454,4 +444,8 @@ export abstract class OdpEventManager implements IOdpEventManager {
   protected getLogger(): LogHandler {
     return this.logger;
   }
+
+  getQueue(): OdpEvent[] {
+    return this.queue;
+  }
```

Comment:
super-NIT: Is it worthwhile to use a getter?

```suggestion
  public get queue(): OdpEvent[] {
     return this.queue;
    }
```

Reply:
I think having the same name for the property and getter will cause problem, and didn't want to rename the property, hence ..